certbot/constants.py

```diff
@@ -32,6 +32,7 @@
     auth_cert_path="./cert.pem",
     auth_chain_path="./chain.pem",
     strict_permissions=False,
+    reuse_key=False,
```

Comment:
+1

This is a new option, so setting it to False to explicitly preserve existing behavior is the best course of action IMHO.

Perhaps I should add a comment to that effect above this line?